test/hotspot/jtreg/gc/arguments/TestArrayAllocatorMallocLimit.java

```diff
@@ -51,7 +51,7 @@ public static void main(String [] args) throws Exception {
   private static final String printFlagsFinalPattern = " *size_t *" + flagName + " *:?= *(\\d+) *\\{experimental\\} *";
 
   public static void testDefaultValue()  throws Exception {
-    ProcessBuilder pb = GCArguments.createTestJvm(
+    ProcessBuilder pb = GCArguments.createLimitedTestJavaProcessBuilder(
```

Comment:
Shouldn't this one read createTestJavaProcessBuilder, as in all other occurences?

Reply:
Hi Lutz,
yes, you are right! But I spoiled this already in the backport to 21, so I'll fix it there in a follow-up.
I files https://bugs.openjdk.org/browse/JDK-8342669 [21u] Fix TestArrayAllocatorMallocLimit after backport of JDK-8315097

Thanks for addressing this tedious review!